<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCardWindow</class>
 <widget class="QDialog" name="AddCardWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(225, 245, 255);
color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QComboBox" name="accountTypeCombo">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>89</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
</string>
   </property>
   <item>
    <property name="text">
     <string>GharzolHasaneh</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>LongTerm</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Jari</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="addCardButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>110</y>
     <width>100</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:5px solid green;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>151</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;add new account&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>240</y>
     <width>100</width>
     <height>33</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:3px solid red;
font: 10pt &quot;Segoe UI&quot;;
background: rgb(255, 255, 255);
color: rgb(0, 0, 0);
gridline-color: rgb(0, 0, 0);
border-radius:10px;</string>
   </property>
   <property name="text">
    <string>close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
